errors/http_status.go

```diff
@@ -14,6 +14,8 @@ func httpStatus(code Code) int {
 		return http.StatusUnauthorized
 	case AuthorizationFailed:
 		return http.StatusForbidden
+	case BasicAuthFailed:
```

Comment:
This case should be merged with `AuthorizationRequired` case.

Reply:
Done